src/users/02-media.md

```diff
@@ -52,16 +52,19 @@ To easily stream videos and audio on Lemmy, you can use any of the following app
 - [Popcorn Time](https://github.com/popcorn-official/popcorn-desktop) (Desktop)
 - [xTorrent](https://play.google.com/store/apps/details?id=com.gamemalt.streamtorrentvideos) (Android)
 
-#### Expert
+#### Advanced
 
-For those who would like to help share files, you can use any of the following torrent clients.
+For those who would like to help share files, you can use any of the following torrent clients:
```

Comment:
Worth mentioning explicitly that you can put the magnet link as post url. And general steps to get the magnet link for a torrent.

Reply:
That's on line 40. But I'll add a note about getting a magnet link for a torrent.